src/php/tests/interop/interop_client.php

```diff
@@ -477,22 +477,27 @@ function statusCodeAndMessage($stub)
     list($result, $status) = $call->wait();
 
     hardAssert($status->code === 2,
-               'Received unexpected status code');
+               'Received unexpected UnaryCall status code: ' .
+               $status->code);
     hardAssert($status->details === 'test status message',
-               'Received unexpected status details');
+               'Received unexpected UnaryCall status details: ' .
+               $status->details);
 
     $streaming_call = $stub->FullDuplexCall();
 
     $streaming_request = new grpc\testing\StreamingOutputCallRequest();
     $streaming_request->setResponseStatus($echo_status);
     $streaming_call->write($streaming_request);
     $streaming_call->writesDone();
+    $result = $streaming_call->read();
```

Comment:
From my understanding, the server is not required to send the initial metadata until it receives the first message from the client? Since PHP is synchronous, if I send the recv_initial_metadata op right from `start()`, `grpc_completion_queue_pluck` will just wait forever. So that's why in the PHP implementation, `start()` will only do send_initial_metadata. Then client can write the first message. Then the first time the client issues a `read()` (and only the first time), it sends the recv_initial_metadata op, along with the recv_message op. 

@murgatroid99 can you please confirm my understanding is correct? Basically, for bidi-streaming, is it true that the PHP client has to issue at least one `read()` to receive the initial metadata?